.gitmodules

```diff
@@ -1,10 +1,12 @@
 [submodule "ceph-object-corpus"]
 	path = ceph-object-corpus
 	url = https://github.com/ceph/ceph-object-corpus.git
+	ignore = dirty
```

Comment:
i'd suggest understand how these unintentional changes were created before ignoring them.

Reply:
git submodule update --init --recursive does solve the issue, but cannot narrow down where it originates, removing this until I observe it again